From 890a1842ce705c56f71ea661d0d0be43e11d5df1 Mon Sep 17 00:00:00 2001
From: tastytea <tastytea@tastytea.de>
Date: Fri, 8 Sep 2023 20:01:03 +0200
Subject: [PATCH] skip some tests

reasons:
  - needs to be in a git repo
  - needs special test dependencies
  - requires network access
  - is time sensitive
---
 common/para/para_test.go                                      | 1 +
 modules/client_test.go                                        | 4 ++++
 .../resource_factories/create/create_integration_test.go      | 1 +
 3 files changed, 6 insertions(+)

diff --git a/common/para/para_test.go b/common/para/para_test.go
index 2d9188e..daf0560 100644
--- a/common/para/para_test.go
+++ b/common/para/para_test.go
@@ -69,6 +69,7 @@ func TestPara(t *testing.T) {
 	})
 
 	c.Run("Time", func(c *qt.C) {
+        c.Skip("time sensitive")
 		const n = 100
 
 		p := New(5)
diff --git a/modules/client_test.go b/modules/client_test.go
index 0ee7e0d..4bb8843 100644
--- a/modules/client_test.go
+++ b/modules/client_test.go
@@ -72,6 +72,7 @@ github.com/gohugoio/hugoTestModules1_darwin/modh2_2@v1.4.0 github.com/gohugoio/h
 	}
 
 	c.Run("All", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		client, clean := newClient(c, func(cfg *ClientConfig) {
 			cfg.ModuleConfig = DefaultModuleConfig
 		}, defaultImport)
@@ -111,6 +112,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("IgnoreVendor", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		client, clean := newClient(
 			c, func(cfg *ClientConfig) {
 				cfg.ModuleConfig = DefaultModuleConfig
@@ -129,6 +131,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("NoVendor", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		mcfg := DefaultModuleConfig
 		mcfg.NoVendor = "**"
 		client, clean := newClient(
@@ -148,6 +151,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("VendorClosest", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		mcfg := DefaultModuleConfig
 		mcfg.VendorClosest = true
 
diff --git a/resources/resource_factories/create/create_integration_test.go b/resources/resource_factories/create/create_integration_test.go
index 1708457..e81f4de 100644
--- a/resources/resource_factories/create/create_integration_test.go
+++ b/resources/resource_factories/create/create_integration_test.go
@@ -25,6 +25,7 @@ import (
 )
 
 func TestGetRemoteHead(t *testing.T) {
+	t.Skip("Requires network access")
 	files := `
 -- config.toml --
 [security]
-- 
2.44.2

